fiat-api/src/main/java/com/netflix/spinnaker/fiat/shared/FiatAuthenticationConfig.java

```diff
@@ -54,22 +56,19 @@
   @Setter
   private RestAdapter.LogLevel retrofitLogLevel = RestAdapter.LogLevel.BASIC;
 
-  @Autowired
-  SpinnakerRequestInterceptor spinnakerRequestInterceptor;
-
   @Bean
   @ConditionalOnMissingBean(FiatService.class) // Allows for override
```

Comment:
I suspect this Conditional was to work around GateConfig explicitly configuring the FiatService and FiatPermissionEvaluator beans. This change removes the need for that duplication